qemu/tests/cfg/virtio_fs_nfs_migration.cfg

```diff
@@ -0,0 +1,120 @@
+    vm_mem_share = yes
+    vm_mem_backend = memory-backend-memfd
+    Win10.i386:
+        mem = 4096
+    !s390, s390x:
+        mem_devs = mem1
+        backend_mem_mem1 = memory-backend-memfd
+        size_mem1 = ${mem}M
+        use_mem_mem1 = no
+        guest_numa_nodes = shm0
+        numa_memdev_shm0 = mem-mem1
+        numa_nodeid_shm0 = 0
+    Windows:
+        # install winfsp tool
+        i386, i686:
+            install_winfsp_path = 'C:\Program Files'
+            devcon_dirname = 'x86'
+        x86_64:
+            install_winfsp_path = 'C:\Program Files (x86)'
+            devcon_dirname = 'amd64'
+        install_winfsp_cmd = 'msiexec /i WIN_UTILS:\winfsp.msi /qn'
+        check_installed_cmd = 'dir "%s" |findstr /I winfsp'
+        viofs_log_file = C:\viofs_log.txt
+        viofs_svc_name = VirtioFsSvc
+        viofs_exe_path = C:\virtiofs.exe
+        viofs_exe_copy_cmd = xcopy %s C:\ /Y
+        viofs_sc_create_cmd = 'sc create ${viofs_svc_name} binpath=${viofs_exe_path} start=auto'
+        viofs_sc_create_cmd += ' depend="WinFsp.Launcher/VirtioFsDrv" DisplayName="Virtio FS Service"'
+        viofs_sc_start_cmd = 'sc start ${viofs_svc_name}'
+        viofs_sc_query_cmd = 'sc query ${viofs_svc_name}'
+        viofs_sc_delete_cmd = 'sc delete ${viofs_svc_name}'
+        debug_log_operation = 'enable'
+        viofs_debug_enable_cmd = 'reg add HKLM\Software\VirtIO-FS /v DebugFlags /d 0xFFFFFFFF /t REG_DWORD'
+        viofs_log_enable_cmd = 'reg add HKLM\Software\VirtIO-FS /v DebugLogFile /d ${viofs_log_file} /t REG_SZ'
+        viofs_debug_delete_cmd = 'reg delete HKLM\Software\VirtIO-FS /v DebugFlags /f'
+        viofs_log_delete_cmd = 'reg delete HKLM\Software\VirtIO-FS /v DebugLogFile /f'
+        viofs_reg_query_cmd = 'reg query HKLM\Software\VirtIO-FS'
+        virtio_win_media_type = iso
+        cdroms += " virtio"
+        cmd_md5 = "%s && md5sum.exe ${test_file}"
+        cmd_dd = 'dd if=/dev/random of=%s bs=1M count=200'
+        guest_file = "%s\${test_file}"
+        fio_options = '--name=fiotest --filename=%s\${test_file}.fio --ioengine=windowsaio --rw=rw '
+        fio_options += '--bs=4k --size=1G --runtime=1800 --time_based --thread'
+        fio_name = fio.exe
+        cmd_chk_fio = 'TASKLIST /FI "IMAGENAME eq FIO.EXE'
+    variants:
+        - cache_mode_auto:
+            fs_binary_extra_options += " --cache auto"
+            fs_binary_extra_options_targetfs += " --cache auto"
+        - cache_mode_always:
+            fs_binary_extra_options += " --cache always"
+            fs_binary_extra_options_targetfs += " --cache always"
+        - cache_mode_never:
+            fs_binary_extra_options += " --cache never"
+            fs_binary_extra_options_targetfs += " --cache never"
+        - cache_mode_metadata:
+            fs_binary_extra_options += " --cache metadata"
+            fs_binary_extra_options_targetfs += " --cache metadata"
+    variants:
+        - @default:
+        - writeback:
```

Comment:
Hi, "With --writeback, most FIO requests will stay in the guest’s page cache, dirtying a lot of memory. This leads to the quite long reported expected migration downtime"  from https://issues.redhat.com/browse/RHEL-29027, so we need to set the downtime-limit to a longer time before migration like:
(qemu) migrate_set_parameter downtime-limit 30000

Thanks!

Reply:
Change done